### DISCUSSION TITLE
Why do Hints and other components not look the same as they do in Gitbook Docs?

### DISCUSSION THREAD

**Discussion by helloimalemur** [GitbookIO/community#595] [2. Published content] [answered]
Why do Hints and other components not look the same as they do in Gitbook Docs?
I am currently on a trial for the "Pro" plan.

![Screenshot from 2024-04-25 12-42-35](https://github.com/GitbookIO/community/assets/37273704/f01e09a0-d1c8-432b-98c8-02a49811ad43)
VS
![Screenshot from 2024-04-25 12-43-07](https://github.com/GitbookIO/community/assets/37273704/eb373c95-bb39-477d-916c-22501ff05cee)

ref;
https://docs.gitbook.com/content-editor/blocks/hint

**Comment by addisonschultz** [comment]
Hi @helloimalemur ! We've recently updated the styling for published docs - as you can see they are not yet reflected in the editor! We do plan on making the styles the same in coming updates, so stay tuned!

**Reply by helloimalemur** [reply]
Wonderful! Thank you for the quick response!

**Reply by kkk-gitbook** [reply]
我想在分享页面中实现旧版的样式效果，如何设置？
![image](https://github.com/GitbookIO/community/assets/172463681/3c555a04-d633-4474-b96b-72bff592a328)

**Comment by denisemcg** [comment]
My hints do not look the same when viewed on my published site.
![image](https://github.com/user-attachments/assets/f99e23fc-f93c-49be-b54a-181eee43612f)

**Reply by addisonschultz** [reply, marked as answer]
Hi @denisemcg - thanks for pointing this out, we've still got it on our radar to merge these styles together!

**Comment by aikoii** [comment]
Heya! We’ve changed the way hints look to make them more consistent and visually aligned with the rest of your content. They should look much better now  – we hope you like the update!  😊

Thank you all for your feedback!

Editor:

![CleanShot 2024-12-05 at 16 05 10@2x](https://github.com/user-attachments/assets/74aa7b7f-75f5-4cfb-afdb-93cc3389c8fe)

Published site:

![CleanShot 2024-12-05 at 16 05 20@2x](https://github.com/user-attachments/assets/8e3f1a1b-725d-4212-a5c5-a9516ecf2d94)

**Reply by denisemcg** [reply]
Much better! Thank you. Although I'd prefer the Info background to be a light shade of blue. Maybe it's my screen? It looks grey. The backgrounds in the other hints match their respective icons.
